daemon/algod/api/algod.oas3.yml

```diff
@@ -5611,21 +5676,56 @@
         "responses": {
           "200": {
             "content": {
+              "application/json": {
+                "schema": {
+                  "properties": {
+                    "txn-groups": {
+                      "description": "A result object for each transaction group that was simulated.",
+                      "items": {
+                        "$ref": "#/components/schemas/SimulationTransactionGroupResult"
+                      },
+                      "type": "array"
+                    },
+                    "version": {
+                      "description": "The version of this response object.",
+                      "type": "integer"
+                    },
+                    "would-succeed": {
+                      "description": "Indicates whether the simulated transactions would have succeeded during an actual submission.",
+                      "type": "boolean"
+                    }
+                  },
+                  "required": [
+                    "txn-groups",
+                    "version",
+                    "would-succeed"
+                  ],
+                  "type": "object"
+                }
+              },
               "application/msgpack": {
                 "schema": {
                   "properties": {
-                    "failure-message": {
-                      "description": "\\[fm\\] Failure message, if the transaction would have failed during a live broadcast.",
-                      "type": "string"
+                    "txn-groups": {
+                      "description": "A result object for each transaction group that was simulated.",
+                      "items": {
+                        "$ref": "#/components/schemas/SimulationTransactionGroupResult"
+                      },
+                      "type": "array"
                     },
-                    "missing-signatures": {
-                      "description": "\\[ms\\] Whether any transactions would have failed during a live broadcast because they were missing signatures.",
+                    "version": {
+                      "description": "The version of this response object.",
+                      "type": "integer"
+                    },
```

Comment:
Do we version responses anywhere else? Can you ask for a particular version response? Is this intended as a flag to let clients know something changed?

Reply:
Do we also return the round number it was evaluated against?

Reply:
Genesis hash will be present in each of the transactions, so it's also present in the response.

Right now the round number isn't present but that's probably a good idea to include. I'll think about it more

Reply:
Added round in 88b7182d1